log/sequencer.go

```diff
@@ -346,25 +331,22 @@ func (s Sequencer) SequenceBatch(ctx context.Context, logID int64, limit int, gu
 	stageStart = s.timeSource.Now()
 
 	// Create the log root ready for signing
-	newLogRoot := trillian.SignedLogRoot{
-		RootHash:       merkleTree.CurrentRoot(),
+	seqTreeSize.Set(float64(merkleTree.Size()), label)
+	newLogRoot := &trillian.SignedLogRoot{
+		RootHash: merkleTree.CurrentRoot(),
```

Comment:
FYI: I'm guessing you'll probably need to run `gofmt` here to make Travis happy.